### ISSUE TITLE
ASWG can't login

### ISSUE THREAD

**Issue by PsychoMantis991** [Aquerr/ARMA-Server-Web-Gui#19]
Hi again,

I have tried again, following the steps indicated and when loading the website, the following message appears.
![image](https://github.com/user-attachments/assets/507be5d8-e6aa-438c-a4d5-5ac04f9583a2)

In the log, the following is shown.
![image](https://github.com/user-attachments/assets/4508908f-6e96-459c-a26c-aeec2f28281a)

After changing the default user and password and trying to log in, the log tells me the following.
![image](https://github.com/user-attachments/assets/4d9e50f5-8e20-4f05-8d96-c02aa8422885)

I'm testing on a virtual machine and instead of using the domain name or localhost, I'm using the server's IP. I'm sending you a screenshot of the .properties file and the environment.prod.ts file.
![image](https://github.com/user-attachments/assets/c637e606-c27e-4052-91bd-97ee50a4bacf)
![image](https://github.com/user-attachments/assets/656883fc-541e-4e83-93f0-a668c501502f)

I'm using ubuntu server 24.04

Best regards.

_Originally posted by @PsychoMantis991 in https://github.com/Aquerr/ARMA-Server-Web-Gui/issues/18#issuecomment-2299612310_

**Comment by Aquerr** [comment]
@PsychoMantis991 I suspect the problem is that you don't have `/api/v1` in your `API_BASE_URL`.
Should be `http://192.168.115.131:8085/api/v1`.

**Comment by PsychoMantis991** [comment]
I'll try it and confirm if it works.

Many thanks

**Comment by PsychoMantis991** [comment]
I have modified the file and started the api.

It shows me the same error

**Comment by Aquerr** [comment]
@PsychoMantis991 Hmm... okay... Let's try debug this. 
Look in your browser, in the `developer tools` (F12) => `network tab`. 
What do you get there when you press `login` button?
Verify that the request address is correct.

**Comment by PsychoMantis991** [comment]
![image](https://github.com/user-attachments/assets/a997cfb2-777d-4734-9a9c-84c5cf13cb98)

**Comment by PsychoMantis991** [comment]
YES! its work!
![image](https://github.com/user-attachments/assets/efc428d7-007a-4633-ae5c-1b9b50414d15)

thanks so much

**Comment by Aquerr** [comment]
No problem. Thank you for reporting this issue. I was not aware of that. 😄 Good luck!
(Remember that you have to install the game by yourself)

**Comment by PsychoMantis991** [comment]
Yes!! I'm already doing my tests to set up my server and test the API

**Comment by PsychoMantis991** [comment]
one question, the param aswg.steam.web-api-token... what is it used for?

**Comment by PsychoMantis991** [comment]
I intend to use workshop to download mods, but of course when I try to search for something, the log tells me that permission is denied, and I think that might be why, and I don't know what I should write...

**Comment by PsychoMantis991** [comment]
problem resolv =) i had to create a new steam api key

**Comment by Aquerr** [comment]
Steam api key is used only for getting workshop information.
Steamcmd on the other hand is used by ASWG to download mods. You need to setup your steamcmd so that ASWG has access to it (exec permission). Cheers.

**Comment by PsychoMantis991** [comment]
steamcmd works, the problem is that when it requires steam guard, it doesn't give you the option to enter the steam guard security code.

I've tried with a new account without steam guard, and since I don't have any games, it gives me problems, and to do it with an account like my personal one, it's more complicated.

**Comment by Aquerr** [comment]
I am aware of that. 
ASWG should work well with password only authentication. My personal recommendation is to have separate steam account for that.

When it comes to Steam Guard, I don't have any ideas on how to handle it in `steamcmd` from ASWG at this moment.

**Comment by PsychoMantis991** [comment]
Exactly, the problem with the other account, from what I have been able to prove, is that it is necessary to have purchased a game, since if not, Steam does not allow you to download anything.

Still, great job, I really like the API. Simple and intuitive.
